rdr_service/model/genomics.py

```diff
@@ -84,8 +84,7 @@ class GenomicSetMember(Base):
 
     validatedTime = Column("validated_time", DateTime, nullable=True)
 
-    sampleId = Column('sample_id', String(80),
-                      ForeignKey("biobank_stored_sample.biobank_stored_sample_id"), nullable=True)
+    sampleId = Column('sample_id', String(80), nullable=True)
```

Comment:
The model is changed, but I didn't find the db migration file in this PR. Does this change already applied in DB?

Reply:
Actually, I made the model change in my last PR but then it was decided that we hold on changing this.  I deleted the migration file as we're not needing the FK on this field. But I didn't go back and change this file. The DB doesn't have the foreign key, and shouldn't as of now, so this is really just fixing the change that I had in my last PR.